lib/server-impl.js

```diff
@@ -15,15 +15,10 @@ function createApp (options) {
     const stores = createStores(options);
```

Comment:
`config` created below. It now passes in `options` before creating `config` with it.

But nevermind, it'll be the same result as it is now.